variants/frontend-stimulus/template.rb

```diff
@@ -0,0 +1,61 @@
+source_paths.unshift(File.dirname(__FILE__))
+
+puts "Setting up stimulus.js"
+
+yarn_add_dependencies %w[
+  @hotwired/stimulus
+  @hotwired/stimulus-webpack-helpers
+]
+
+if TEMPLATE_CONFIG.use_typescript?
+  copy_file "variants/frontend-stimulus/hello_controller.ts", "app/frontend/javascript/controllers/hello_controller.ts"
+else
+  copy_file "variants/frontend-stimulus/hello_controller.js", "app/frontend/javascript/controllers/hello_controller.js"
+end
+
+prepend_to_file "app/frontend/packs/application.js" do
+  <<~EO_JS_IMPORTS
+    import { Application } from '@hotwired/stimulus';
+    import { definitionsFromContext } from '@hotwired/stimulus-webpack-helpers';
+  EO_JS_IMPORTS
+end
+
+if TEMPLATE_CONFIG.use_typescript?
+  append_to_file "app/frontend/packs/application.js" do
+    <<~EO_TS_SETUP
+
+      // Set up stimulus.js https://stimulus.hotwired.dev/
+      const application = Application.start();
+      const context = require.context(
+        '../javascript/controllers',
+        true,
+        /.(js|ts)$/u
+      );
+
+      application.load(definitionsFromContext(context));
+
+      // Configure stimulus development experience
+      application.debug = false;
+      // window.Stimulus = application;
+    EO_TS_SETUP
```

Comment:
@G-Rath Again the TS setup is identical to the JS setup here. Is there anything I should add/change?

Reply:
Nope, though it should be `application.ts`

Reply:
I'll fix that. 🤔 I think I'll need to look at the other TS variants too because they'll need to check that some other variant hasn't already renamed the file before they try to do it.